### DISCUSSION TITLE
Not able to Install on Linux Machine

### DISCUSSION THREAD

**Discussion by akash-goel** [rilldata/rill#3192] [General]
Hi Team,

I am trying to install Rill Developer using the below command

`curl -s https://cdn.rilldata.com/install.sh | bash` I am getting the error "Unknown Option = ignore-missing" 

![image](https://github.com/rilldata/rill/assets/5937612/dc61b3f2-8ed5-46d4-8bd7-08b3bbb1d1b9)

Can you please suggest , Is rill data work on specific Linux version or what we can do to resolve the issue ?

Regards
Akash

**Comment by akash-goel** [comment]
Hi Team , Request you to suggest the correct way of install the Rill developer on window or linux machine

**Comment by k-anshul** [comment]
Hi @akash-goel 

We don't support windows yet. What's your specific linux version and distribution ?

**Comment by akash-goel** [comment]
Hi @k-anshul 

we are using UbuntuServer , 18_04-lts-gen2

**Comment by k-anshul** [comment]
Hi @akash-goel 

Given 18.04 is EOL we recommend you to upgrade to 22.04 which is also the most recent LTS version

**Comment by akash-goel** [comment]
Hi Anshul ,

Can you please confirm which Centos version Rill Support , we can try that , will it support 7.5/7.6/7.7 versions or any other version it will support?

**Comment by k-anshul** [comment]
Hey

We haven't tried on Centos so you can try if its supported there. Looking at the Centos version it seems like 7.x is also a fairly old release. Would recommend to try on latest version if possible.
